utilities/externalModPhase/src/main.ts

```diff
@@ -0,0 +1,60 @@
+import { config } from "dotenv";
```

Comment:
It could be helpful to have a short README about how to use this testing utility for future reference.